deform_ext_autocomplete/__init__.py

```diff
@@ -75,7 +75,7 @@ def serialize(self, field, cstruct, readonly=False):
         if cstruct in (null, None):
             cstruct = ''
         options = {}
-        if not self.delay:
+        if not hasattr(self, 'delay'):
```

Comment:
I do get an error about `delay` with the current master branch that is resolved by one of your commits. So it looks good.

However, I also get an error about a missing `size` attribute when running the demo application (`demo/app.py` in this project, slightly modified to be compatible with Python 3) in brand new virtual env (thus with Deform 2):

```
    self._render(stream, econtext, rcontext)
  File "ext_autocomplete_input_6d99725de9b5995ce9487818ed436e37.py", line 360, in render
  File "/home/damien/env/deform_ext_autocomplete/lib/python3.8/site-packages/chameleon/py26.py", line 11, in lookup_attr
    raise exc
  File "/home/damien/env/deform_ext_autocomplete/lib/python3.8/site-packages/chameleon/py26.py", line 5, in lookup_attr
    return getattr(obj, key)
AttributeError: 'ExtendedAutocompleteInputWidget' object has no attribute 'size'

 - Expression: "field.widget.size"
 - Filename:   ... orm_ext_autocomplete/templates/ext_autocomplete_input.pt
 - Location:   (line 9: col 28)
 - Source:     tal:attributes="size field.widget.size;
                                    ^^^^^^^^^^^^^^^^^
```

Don't you have this error too? Or perhaps you have customized the template?

This is probably because this project was made for Deform 1. I am willing to tackle that by adapting what's needed for Deform 2 and the latest version of jQUery UI (and also removing Python 2 compatibility). Would that be fine? You're probably one of the only users, so I would not want to lose you if you're using less modern versions. :)

If that does not suit you, I would propose to accept your pull request, cut a new release, and only then do the aforementioned cleaning.

Reply:
Hi @dbaty!

Python 2? Ew, no. Your proposed changes sound good, but let me take a look at what we've got on our project and tidy up this PR. It's one of those where we made the change to get it working but never took a step back and checked everything - I should have done that before the PR really...

Cheers, I'll get you an update within a week.

Matt